scripts/download-pokemon-data.js

```
@@ -0,0 +1,342 @@
+/**
+ * Download Pokemon TCG data from GitHub
+ * This script downloads all sets and cards data from the Pokemon TCG GitHub repository
+ * and saves them to the public/data directory for local access.
+ *
+ * It uses a caching strategy to avoid downloading data that hasn't changed,
+ * which is important for frequent builds.
+ */
+
+const fs = require('fs');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)